source/_integrations/squeezebox.markdown

```diff
@@ -46,6 +49,15 @@ transporter_toslink:
         media_content_type: "music"
 ```
 
+## Entities
+
+### Binary sensors
+
+- Needs restart
+  Sever Service would like to be restarted usually to apply updates.
+- Library rescan
+  Music library is currently being scaned by LMS.
```

Comment:
does this have implications for me as a user? is some service not available during that period?